hackclub.com.yaml

```diff
@@ -1610,6 +1610,10 @@ malagon:
   - ttl: 600
     type: CNAME
     value: hackclubmalagon.netlify.app.
+mangu:
+  - ttl: 600
+    type: CNAME
+    value: https://mangu.pages.dev.
```

Comment:
Hi there,

CNAME records cannot have https:// or any other protocol prefix, could you remove it? 

Also make sure the custom domain is configured on Cloudflare pages.